include/vk/vkwrap/surface.h

```diff
@@ -12,13 +14,10 @@ class Surface : private vk::UniqueSurfaceKHR
   private:
     using BaseType = vk::UniqueSurfaceKHR;
 
-  private:
-    // Call permissions: any thread.
-    static vk::SurfaceKHR createSurface( vk::Instance instance, GLFWwindow* window );
-
   public:
-    Surface( vk::Instance instance, GLFWwindow* window )
-        : BaseType{ createSurface( instance, window ) }
+    // Call permissions: any thread.
+    Surface( vk::Instance instance, wnd::Window& window )
+        : BaseType{ wnd::Window::createWindowSurface( instance, window ) }
```

Comment:
This class still depends on the window wrapper. If the ```wnd::Window&``` were an interface class, then this would be correct. But...
Why not something like:
```cpp
Surface( vk::UniqueSurfaceKHR surface )
        : BaseType{ std::move(surface); }
```

Reply:
Our `wnd::Window` class provides platform agnostic API. So, if you changed GLFW to SDL, you would implement only body of `wnd::Window::createWindowSurface`, not its interface.